django/contrib/admin/checks.py

```diff
@@ -607,8 +607,8 @@ def _check_readonly_fields_item(self, obj, field_name, label):
             except FieldDoesNotExist:
                 return [
                     checks.Error(
-                        "The value of '%s' is not a callable, an attribute of '%s', or an attribute of '%s.%s'." % (
```

Comment:
I didn't make any cosmetic edits (as wrapping at 79 chars) to keep diff as small as possible.

Reply:
We're touching this line. I'd be inclined to fix it...

Reply:
OK, wrapped.